src/test/java/org/fcrepo/vocabulary/AbstractVocabularyTest.java

```diff
@@ -47,6 +50,10 @@
 
     public abstract Class vocabulary();
 
+    static {
+        setDefaultHttpClient(create().setRedirectStrategy(new LaxRedirectStrategy()).build());
```

Comment:
This seems reasonable, but maybe you want to build in some caching and pooling behavior? I'm not sure what the defaults are.